interface/main/tabs/templates/user_data_template.php

```diff
@@ -14,10 +26,11 @@
                     <?php } ?>
                     <li data-bind="click: userPrefs"><?php echo xlt("User Preferences");?></li>
                     <li data-bind="click: changePassword"><?php echo xlt("Change Pass Phrase");?></li>
```

Comment:
Another thing to look for is php tags crammed up against other stuff.  It is fine here, but there are cases where it breaks stuff (like in query builders).